docs/content/docs/concepts/signals.md

```diff
@@ -6,6 +6,95 @@ discussion: 5469
 aliases:
   - "/docs/advanced/signals/"
   - "/docs/deep/signals/"
+draft: true
 ---
 
-Signals are similar to ops, but are not stored; they're useful to implement features like user presence.
+Signals are similar to ops, but are not stored; they're useful to implement features like user presence. They provide a simple way to communicate transient data to connected clients within a Fluid application.
+
+They are lossy and not guaranteed to be ordered on delivery relative to other signals and ops. A good analogy to help conceptualize signals is that they can be interpreted as UDP protocol, whereas standard ops can be thought of as TCP protocol.
+
+## Why are signals useful?
```

Comment:
I think the audience for this article will likely be someone who knows how to use DDSs, but has not yet used signals.  So, this section might benefit from framing it as a comparison against storing the information in a DDS -- instead of just "signals don't do X" it can be "using a DDS does X, but signals don't".

E.g. "When you use a DDS to communicate data, that data is retained as part of the container.  For data that is transient this storage can be wasteful.  By contrast, data communicated via signals is not retained, making them a good fit for transient data."